cou/apps/app.py

```diff
@@ -270,12 +284,13 @@ def current_os_release(self) -> OpenStackRelease:
         :return: OpenStackRelease object
         :rtype: OpenStackRelease
         """
-        os_versions = {unit_values["os_version"] for unit_values in self.units.values()}
+        os_versions = {unit.os_version for unit in self.units}
 
         if len(os_versions) == 1:
             return os_versions.pop()
         # NOTE (gabrielcocenza) on applications that use single-unit or paused-single-unit
         # upgrade methods, more than one version can be found.
+        # return OpenStackRelease("ussuri")
```

Comment:
I think this was used for debugging purpose, or not?

Reply:
done